charts/images.yaml

```diff
@@ -3,9 +3,9 @@ images:
   resourceId:
     name: 'etcdbrctl'
   sourceRepository: github.com/gardener/etcd-backup-restore
-  repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
-  tag: "v0.24.1"
+  repository: eu.gcr.io/gardener-project/gardener/etcdbrctl #aaronfernandes/etcdbr #eu.gcr.io/gardener-project/gardener/etcdbrctl
+  tag: "v0.25.0-dev-2f39d84b1ad90207f1f825d74f896132bf6f20df" #"wrap_test_3" #"v0.24.1"
```

Comment:
before merging the PR, don't forget to update the image tag and remove this comment